src/main/controllers/rpc-handler.js

```diff
@@ -751,7 +752,7 @@ module.exports = function(app) {
 									kycprocess,
 									requiredStaticData[i]._id
 								);
-								console.log('THE static data', staticDatas);
+								log.debug('THE static data %s', staticDatas);
```

Comment:
Are we sure that this doesn't contain any private info? Now that we're working on integrating Sentry as a log transport we need to be very careful when logging variables.

Reply:
I believe you are partially right. I think we should strongly define log levels and what log levels will be enabled by default for users in deferent modes (normal and issue reproduction). In any case I think debug and trace should be available only to developers so we should not be concerned with security for thous.